# kmers-project

## Project GENOM 2021-2022
Global Analysis of Prokaryotic Genomes by Word Distribution

Damien Legros, Cédric Cornède, and Valentin Gherdol

In this project, we focused on the analysis of K-mers within bacterial and archaeal genomes.

**References used :**
- Sims G. E., Jun S. R., Wu G. A. and Kim S. H. (2009). Alignment -free genome comparison with feature frequency profiles (FFP) and optimal resolutions. Proc Natl Acad Sci U S A 106, 2677-2682.
- Deschavanne P., DuBow M. S. and Regeard C. (2010). The use of genomic signature distance between bacteriophages and their hosts displays evolutionary relationships and phage growth cycle determination. Virol J 7, 163.
- Deschavanne P. and Tuffery P. (2008). Exploring an alignment free approach for protein classification and structural class prediction. Biochimie 90, 615-625.
- Chapus C., Dufraigne C., Edwards S., Giron A., Fertil B. and Deschavanne P. (2005). Exploration of phylogenetic data using a global sequence analysis method. BMC Evol Biol 5, 63.
- Deschavanne P. J., Giron A., Vilain J., Fagot G. and Fertil B. (1999). Genomic signature: characterization and classification of species assessed by c haos game representation of sequences. Mol Biol Evol 16, 1391-1399.
